### ISSUE TITLE
A request for inclusion of RP-1 Legacy Version Chinese wiki and installation tutorial.

### ISSUE THREAD

**Issue by KEKKJ123** [KSP-RO/RP-1#2431]
**I have read the contribution guide for this repository.**
---
Hello, I am a Chinese RP-1 player. I like this mod, so I wrote a mod introduction and installation tutorial about Legacy Version for Chinese on my blog website and Bilibili. I hope my article can be included, and I really appreciate your efforts.
My English is so poor that I have to use machine translation. I'm sorry about this and thank you again.
The blog link is here: https://kekkj123.github.io/posts/3/
Bilibili: https://www.bilibili.com/read/cv18962113/
![1](https://github.com/user-attachments/assets/e184e056-7395-47ad-8804-9bb92e4318b3)
![2](https://github.com/user-attachments/assets/0c6ffe61-524b-4c38-aed5-d5188c1596fd)

**Comment by Clayell** [comment]
Bilibili link seems to not work. First link also did not work, but I found a replacement at https://kekkj123.github.io/posts/Legacy_RP1_Installation_Guide/. I will add this to https://github.com/KSP-RO/RP-1/wiki/Installation.

**Comment by KEKKJ123** [comment]
I'm sorry I forgot to modify the wrong blog URL. The Bilibili link should now be accessible. 
And here is another article introducing RP-1 Legacy.https://kekkj123.github.io/posts/RSSRORP1_Seris_Introduction/

**Comment by Clayell** [comment]
Yep, links seem to work now. I'll put the new links next to the other one.
